R/data_export.R

```diff
@@ -60,7 +60,7 @@ export_data <- function(..., database, URL) {
 
     #Adding static source attributes to each dataset
     attr(env[[database]][[dataset_name]], "source_URL") <- URL
-    attr(env[[database]][[dataset_name]], "source_bib") <- bibtex::read.bib(file = paste0("data-raw/",
+    attr(env[[database]][[dataset_name]], "source_bib") <- RefManageR::ReadBib(file = paste0("data-raw/",
```

Comment:
Does this have precisely the same output?

Reply:
Yes, in this case they do. This was done to fix issues with suggesting/relying on the "bibtex" package which is currently not being maintained. The function also calls the "bibtex" package btw.